packages/d3fc-series/src/webgl/bar.js

```diff
@@ -7,7 +7,7 @@ export default () => {
 
     const draw = glBar();
 
-    const bar = (data) => {
+    const bar = data => {
```

Comment:
In the other WebGL series implementations we've got brackets around the data parameter, probably best that this stays consistent

Reply:
Do we have a prettier config? This file was auto-formatted in my editor.

Reply:
See my previous comment. The configuration is stored in the `eslintrc.json` file.

Reply:
Done.